docs/qa/v037/CometBFT.md

```diff
@@ -0,0 +1,172 @@
+
+### Saturation point
+
+As previous iterations, in our QA experiments, the system is subjected to a load slightly under a saturation point.
+The method to identify the saturation point is explained [here](../v034/README.md#finding-the-saturation-point) and its application to the baseline is described [here](./TMCore.md#finding-the-saturation-point).
+We use the same saturation point, that is, `c`, the number of connections created by the load runner process to the target node, is 2 and `r`, the rate or number of transactions issued per second, is 200.
+
+## Examining latencies
+
+The following figure plots six experiments carried out with the network.
+Unique identifiers, UUID, for each execution are presented on top of each graph.
+
+![latencies](./img/200nodes_cmt037/all_experiments.png)
+
+We can see that the latencies follow comparable patterns across all experiments.
+Therefore, in the following sections we will only present the results for one representative run, chosen randomly, with UUID starting with `75cb89a8`.
+
+![latencies](./img/200nodes_cmt037/e_75cb89a8-f876-4698-82f3-8aaab0b361af.png).
+
+For reference, the following figure shows the latencies of different configuration of the baseline.
+`c=02 r=200` corresponds to the same configuration as in this experiment.
+
+![all-latencies](./img/200nodes_tm037/v037_200node_latencies.png)
+
+As can be seen, latencies are similar.
+
+## Prometheus Metrics on the Chosen Experiment
+
+This section further examines key metrics for this experiment extracted from Prometheus data regarding the chosen experiment.
+
+### Mempool Size
+
+The mempool size, a count of the number of transactions in the mempool, was shown to be stable and homogeneous at all full nodes. 
+It did not exhibit any unconstrained growth.
+The plot below shows the evolution over time of the cumulative number of transactions inside all full nodes' mempools at a given time.
+
+![mempoool-cumulative](./img/200nodes_cmt037/mempool_size.png)
+
+The following picture shows the evolution of the average mempool size over all full nodes, which mostly oscilate between 1500 and 2000 outstanding transactions.
+
+![mempool-avg](./img/200nodes_cmt037/avg_mempool_size.png)
+
+The peaks observed coincide with the moments when some nodes reached round 1 of consensus (see below).
+
+
+The behavior is similar to the observed in the baseline, presented next.
+
+![mempool-cumulative-baseline](./img/200nodes_tm037/v037_r200c2_mempool_size.png)
+
+![mempool-avg-baseline](./img/200nodes_tm037/v037_r200c2_mempool_size_avg.png)
+
+
+### Peers
+
+The number of peers was stable at all nodes.
+It was higher for the seed nodes (around 140) than for the rest (between 16 and 78).
+The red dashed line denotes the average value.
+
+![peers](./img/200nodes_cmt037/peers.png)
+
+Just as in the baseline, shown next, the fact that non-seed nodes reach more than 50 peers is due to #9548.
```

Comment:
```suggestion
Just as in the baseline, shown next, the fact that non-seed nodes reach more than 50 peers is due to [tendermint/tendermint\#9548](https://github.com/tendermint/tendermint/issues/9548).
```

Do we have a corresponding issue in the CometBFT repo? The original one was automatically closed by stalebot.

Reply:
We don't.
Using the references at the end.